lava-job-runner.py

```diff
@@ -1,4 +1,22 @@
 #!/usr/bin/python
+#
+# Copyright (C) 2014, 2015, 2016 Linaro Limited
```

Comment:
I'm not one of the founders of kci so I'd want Tyler, Milo or Kevin to approve the license changes before I'd merge this